components/schemas/Metric.yaml

```diff
@@ -0,0 +1,17 @@
+type: object
+properties:
+  _links:
+    $ref: ./Links.yaml
+  _id:
+    $ref: ./Id.yaml
+  key:
+    type: string
+    example: test-metric
+  name:
+    type: string
+    description: Name of the metric.
+    example: Test Metric
+  kind:
+    type: string
+    description: Whether the metric is a click, pageview or custom metric.
```

Comment:
Are there only three possible values? If so, you can add an enum definition:

```
enum:
 - click
 - pageview
 - custom
```

If however, custom metric means they can put any string value here, then ignore me... 😆

Reply:
Your suggestion will work, and is a good improvement. thanks for letting me know. custom is still the value for the metric type, and its value is separately defined.